doc/api/events.md

```diff
@@ -1661,12 +1661,24 @@ console.log(listenerCount(myEmitter, 'event'));
 added:
  - v13.6.0
  - v12.16.0
+changes:
+  - version:
+    - v20.0.0
+    pr-url: https://github.com/nodejs/node/pull/41276
+    description: The `close`, `highWatermark`, and `lowWatermark`
+                 options are supported now.
 -->
 
 * `emitter` {EventEmitter}
 * `eventName` {string|symbol} The name of the event being listened for
 * `options` {Object}
   * `signal` {AbortSignal} Can be used to cancel awaiting events.
+  * `close` - {string\[]} Can be used to set event names that will end
+    the iteration.
+  * `highWatermark` - {integer} **Default:** `Number.MAX_SAFE_INTEGER`
+    Sets the `highWaterMark` used for the stream.
+  * `lowWatermark` - {integer} **Default:** `1`
+    Sets the `lowWaterMark` used for the stream.
```

Comment:
I am pretty sure there is no stream involved here.

these comments seem better:
https://github.com/nodejs/node/blob/2aaa21f9f684484edb54be30589c4af0b923cdef/lib/events.js#L1135
https://github.com/nodejs/node/blob/2aaa21f9f684484edb54be30589c4af0b923cdef/lib/events.js#L1141